ext/zip/config.m4

```diff
@@ -1,93 +1,22 @@
 dnl config.m4 for extension zip
 
 PHP_ARG_ENABLE([zip],
-  [for zip archive read/writesupport],
+  [for zip archive read/write support],
   [AS_HELP_STRING([--enable-zip],
     [Include Zip read/write support])])
 
-PHP_ARG_WITH([libzip],
-  [libzip],
-  [AS_HELP_STRING([[--with-libzip[=DIR]]],
-    [ZIP: use libzip])],
-  [yes],
-  [no])
-
 if test "$PHP_ZIP" != "no"; then
+  PKG_CHECK_MODULES([LIBZIP], [libzip >= 1.3.1])
```

Comment:
We can't bump the version this far, even 18.04 only has libzip 1.1.2. We'll have to keep some of the detection code.

Reply:
Oh, sorry. I run Debian Sid, which has libzip 1.5.1 already. Are you happy to set 1.1.2 as the minimum required version?

Reply:
It's probably best to stick with the existing requirement of 0.11 -- we should discuss bumping the version separately from pkg-config migration.

Reply:
Sure. I'll this code back in.